torch/csrc/distributed/rpc/init.cpp

```diff
@@ -39,21 +39,25 @@ PyObject* rpc_init(PyObject* /* unused */) {
       shared_ptr_class_<RpcBackendOptions>(
           module,
           "RpcBackendOptions",
-          R"(A structure encapsulating the options passed into the RPC backend.
-            An instance of this class can be passed in to :meth:`~torch.distributed.rpc.init_rpc`
-            in order to initialize RPC with specific configurations, such as the
-             RPC timeout and init_method to be used. )")
+          R"(An abstract structure encapsulating the options passed into the RPC
```

Comment:
We can't directly pass in `RpcBackendOptions`, correct? Should we explicitly state this, and maybe remove `RpcBackendOptions` from the public docs? It doesn't seem like the user will ever have to construct this - they'd always use the process group variant.

Reply:
Right, we cannot pass `RpcBackendOptions` directly, let me remove it then.